File: StreamChampion/src/main/java/com/streamchampion/resources/database/Connection.java
package com.streamchampion.resources.database;

import java.sql.DriverManager;
import java.sql.SQLException;

public class Connection {

    private String hostName = "servercarbontower.database.windows.net";
    private String dbName = "carbontower";
    private String username = "tower";
    private String password = "!Carbon6";
    private String url = String.format("jdbc:sqlserver://%s:1433;database=%s;user=%s;password=%s;encrypt=true;"
            + "hostNameInCertificate=*.database.windows.net;loginTimeout=30;", hostName, dbName, username, password);
    private java.sql.Connection connection = null;

    public java.sql.Connection getConnection(){
        try{
            connection = DriverManager.getConnection(url);
            String schema = connection.getSchema();
            System.out.println("Successful connection - Schema: " + schema);

            return connection;

        }catch(SQLException e){
            System.out.println(e);
        }

        return connection;
    }





}

File: teste-http/src/main/kotlin/Connection.kt
import org.jetbrains.exposed.sql.Database

object Connection{

    fun connectWithSqlServer(
        server: String,
        databaseName: String,
        databaseUser: String,
        databasePassword: String
    ){
        Database.connect(
            url = "jdbc:sqlserver://${server};databaseName=${databaseName}",
            driver = "com.microsoft.sqlserver.jdbc.SQLServerDriver",
            user = databaseUser,
            password = databasePassword
        )
    }

    fun connectWithCarbonTowerDatabase(){
        connectWithSqlServer(
            server = "servercarbontower.database.windows.net",
            databaseName = "carbontower",
            databaseUser = "tower",
            databasePassword = "!Carbon6"
        )
    }

}